index.html

```diff
@@ -2,45 +2,220 @@
     <link href='https://fonts.googleapis.com/css?family=Playfair+Display' rel='stylesheet'>
     <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
     <title>Felix Odete's Portfolio</title>
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta content="width=device-width, initial-scale=1" name="viewport">
 </head>
 <body>
-    <header class="head-container">
-        <a href="#" class="home-logo">
-            <div class="logo">
-                <img src="img/yellow-triangle.svg" alt="" id="yellow-triangle">
-                <img src="img/red-square.svg" alt="" id="red-square">
+<header class="head-container">
+    <a class="home-logo" href="#">
+        <div class="logo">
+            <div class="yellow-triangle"></div>
+            <div class="red-square"></div>
+        </div>
+    </a>
+    <nav class="nav flex-container">
+        <a href="#">
+            <div class="menu-button">
+                <div class="rectangle"></div>
+                <div class="rectangle"></div>
+                <div class="rectangle"></div>
             </div>
         </a>
-        <nav class="nav flex-box">
-            <a href="#">
-                <div class="menu-button">
-                    <div class="rectangle"></div>
-                    <div class="rectangle"></div>
-                    <div class="rectangle"></div>
-                </div>
-            </a>
-        </nav>
-    </header>
-    <div class="container">
-        <main>
-            <section>
-                <div class="name">
-                    <h1>Felix Odete</h1>
-                </div>
-                <div class="about-me">
-                    <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
-                        If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
-                </div>
+    </nav>
+</header>
+    <main class="flex-container">
+        <div>
+            <div class="name">
+                <h1>Felix Odete</h1>
+            </div>
+            <div class="about-me flex-container">
+                <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of
+                    my works.
+                    If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+            </div>
+            <a href="">
                 <div class="rectangle-button">
                     <h1>Start collaboration</h1>
                 </div>
-            </section>
-        </main>
-    </div>
+            </a>
+            <div class="middle-image"></div>
+        </div>
+        <div>
+            <h2 class="projects">Projects</h2>
+        </div>
+        <div class="computer-image"></div>
+        <div class="project-section">
+            <h2>Project name goes here</h2>
+        </div>
+        <div class="grid-container">
```

Comment:
- [ ] [Optional] - I suggest you make your CSS dry by not repeating the same CSS style rules for grid-container, grid-container-1, grid-container-2, grid-container-3, grid-container-4, grid-container-5